### DISCUSSION TITLE
Does Copier 7.0.0 have any breaking changes at the CLI level? How about API level?

### DISCUSSION THREAD

**Discussion by blakeNaccarato** [copier-org/copier#841] [Q&A] [answered]
I appreciate the continued effort on this amazing tool! I've been using it since I heard about it in Jairo's appearance on `podcast.__init__`.

I see that Copier has reached 7.0.0. Does this major version bump indicate any breaking changes for users of the CLI alone? Or are the breaking changes confined to the Python API? Or perhaps it's just a significant refactor worthy of a v7?

**Comment by yajo** [comment, marked as answer]
Only minor refactors at API level, and 7.0.1 (which should have been included in 7.0.0) drops support for old bracket template markers by default.

Possibly a mistake on my side on what was considered to bump a big number. I use currently commitizen for the tag and release, and it seems some commits were marked as `refactor`, which in turn bumps the major version.

**Reply by blakeNaccarato** [reply]
No worries, thanks for clarification! When you have a tool like `copier` which exposes both the CLI and Python API, a refactor in either seems worth a major bump. It's all semantics, anyways, I was just checking in to see if I should expect any major gotchas in migrating from v6 to v7.

Thanks!
